gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -722,6 +849,10 @@ public String getEndpoint() {
       return endpoint;
     }
 
+    public EndpointContext getEndpointContext() {
+      return endpointContext;
+    }
```

Comment:
Same with this. Can this either be removed or be made package-private?

Reply:
No this is not being used anywhere, added it for consistency, but I agree that we should avoid exposing the getter for this. Removed in https://github.com/googleapis/sdk-platform-java/commit/33bd7a6be4f8ab28621956a80f1f166893591408